lib/api-create.js

```diff
@@ -18,17 +20,26 @@ module.exports = class ClientCreateAPI extends API {
 
 	async process({ clients: clientCodes } = this.data) {
 
-		const clientsToCreate = await ModelClient.formatForCreate(clientCodes);
-
 		const ClientModel = ModelFetcher.get(); // se tiene que usar el modelo del servicio
-
 		const model = new ClientModel();
 
+		const clients = ClientModel.additionalFields ? await this.getClients(clientCodes) : clientCodes.map(code => ({ code }));
+
+		if(!clients) {
+			logger.error('Unable to get Janis ID clients, they won\'t be created.');
+			return;
+		}
+
+		if(clients.length !== clientCodes.length)
+			logger.warn('Some clients couldn\'t be get from Janis ID service, they won\'t be created.');
```

Comment:
couldn't be tendria que ir con el participio. Igual get queda rari. Que opinas de "couldn't be obtained/retreived"